unkeyed/unkey
# chore: use knip to remove unused files and deps

## What does this PR do?

- Adds https://knip.dev/ to the monorepo root as a dependency along with a config file and run script
- Removes files and dependencies flagged as unused by that script, removing 26 unused files, 53 unused dependencies, and 22 unused dependencies.
- This will speed up install and build times of our apps while improving code health and maintainability

## Type of change


## How should this be tested?

- Run pnpm build from the monorepo root to ensure that all builds pass with removed dependencies

## Checklist
